diff -Nru mailcap-2.1.54.p1/mailcap mailcap-2.1.54.p2/mailcap
--- mailcap-2.1.54.p1/mailcap	2026-08-02 22:03:41.499584234 +0200
+++ mailcap-2.1.54.p2/mailcap	2026-08-02 22:03:41.526495932 +0200
@@ -12,6 +12,25 @@
 
 application/x-bittorrent; /usr/bin/xdg-open %s
 
+# openoffice
+application/vnd.oasis.opendocument.text; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.chart; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.chart-template; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.formula; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.formula-template; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.graphics; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.graphics-template; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.image; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.image-template; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.presentation; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.presentation-template; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.spreadsheet; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.spreadsheet-template; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.text; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.text-master; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.text-template; ooffice %s; needsterminal
+application/vnd.oasis.opendocument.text-web; ooffice %s; needsterminal
+
 text/html; /usr/bin/xdg-open %s ; copiousoutput
 
 type=application/x-java-jnlp-file; description="Java Web Start"; exts="jnlp"
